### ISSUE TITLE
Bad subkey when following someone?

### ISSUE THREAD

**Issue by dhhdev** [keybase/keybase-issues#2659]
So, proving accounts and doing everything else seems to work as I wanted/is intended, but when I follow somebody I get this error:

```bash
WARNING Bad subkey: openpgp: invalid data: Subkey wasn't signed; expected a 'binding' signature```
```

I can still follow people, nothing changes there. But the error message has me scratching my head.

The only thing I've changed is:

```bash
keybase config set gpg.command /usr/bin/gpg2 
```
(Since I have both gpg and gpg2 on my Linux machine.)

My GPG is the one I use normally, I just ran ```keybase select```. Maybe it has something to do with following, on their end and not mine?

**Comment by maxtaco** [comment]
Who was it?

**Comment by dhhdev** [comment]
Oh, sorry. Silly me not saying that. In this case it was ekkis. :+1:

**Comment by maxtaco** [comment]
Thanks. Does it work for other people?

**Comment by dhhdev** [comment]
Yes it does, I tried out: ```keybase follow max``` and no errors. That pretty much settles it I guess. :+1:

**Comment by dhhdev** [comment]
I can also add that this happens when encrypting a message to the said user etc. So yeah, I guess it's a problem on his side.

**Comment by maxtaco** [comment]
Ah, ok thanks for the feedback.  Please note that this is an error that can be safely ignored, it is probably caused by a stale PGP subkey, which otherwise is needed for correct identification purposes.

**Comment by dhhdev** [comment]
Ah, okay. Maybe having a more "human" error in this case, would be awesome to have. (Or warning rather warning.) Keep up the good work, though. Love the project. 👍  Works like a charm so far, at least. :-)
